app/controllers/application_controller.rb

```diff
@@ -64,7 +62,7 @@ def not_found
   end
 
   def prohibited
-    render '/public/prohibited.html', layout: false
+    render '/public/403.html', layout: false
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.